<launch>
    <!-- <param name="use_sim_time" value="true" /> -->
   
    <node pkg="aunchan_core" type="imu_attitude_to_tf" name="imu_attitude_to_tf">
        <remap from="imu_topic" to="attitude" />
        <param name="base_stabilized_frame" type="string" value="base_stabilized" />
        <param name="base_frame" type="string" value="base_link" />
    </node>
    <!-- <node pkg="tf" type="static_transform_publisher" name="base_stabilized_to_base_link"
        args="0.0 0.0 0.0 0.0 0.0 0.0 base_stabilized base_link 100"/> -->

    <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser"
        args="0.0 0.0 0.1 0.0 0.0 0.0 base_link laser 100"/>
    
    <node pkg="tf" type="static_transform_publisher" name="base_link_to_imu"
        args="-0.02 -0.02 0.01 0.0 0.0 0.0 base_link imu_frame 100"/>

    <node pkg="tf" type="static_transform_publisher" name="base_stabilized_to_base_footprint"
        args="0.0 0.0 0.082 0.0 0.0 0.0 base_footprint base_stabilized 100"/>

</launch>